extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -399,17 +417,19 @@ public function categoryTopArticles() {
 						break;
 					}
 				}
-				if ( ! empty( $ids ) ) {
+				if ( !empty( $ids ) ) {
 					$params = [ 'ids' => implode( ',', $ids ), 'height' => 50, 'width' => 50, 'abstract' => 150 ];
-					$detailResponse = $this->app->sendRequest( 'ArticlesApiController', 'getDetails', $params )->getData();
+					$detailResponse =
+						$this->app->sendRequest( 'ArticlesApiController', 'getDetails', $params )->getData();
 					foreach ( $detailResponse['items'] as $item ) {
 						$processed = static::processArticleItem( $item );
 						if ( !empty( $processed ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)